### ISSUE TITLE
Expected error or warning when attempting route with null or invalid `page`

### ISSUE THREAD

**Issue by kcjonson** [redfin/react-server#104]
Ran into this one pretty quick, I had a typo in the `page` attribute in my routes file and the log didn't give any warning that the router was unable to find a valid page class at that filesystem location.

Log output: 
`2016-04-19T22:05:36.031Z - debug: [react-server.core.context.Navigator] Navigating to /home` 

Route:
```
routes: {
	Test: {
		path: ["/home"],
		page: "./pages/Hone"
	},
}
```

Note: it should be "Home" not home, as the page class is at `./pages/Home.js`

I expected some sort of warning here in the log, particularly when not running in prod mode.

**Comment by gigabo** [comment]
Related to #95.  Also addressed by #80.

**Comment by gigabo** [comment]
Just landed #80.
